src/ontology/uberon-edit.obo

```diff
@@ -61074,6 +61071,7 @@ xref: FMA:72595
 xref: HBA:265504914
 xref: neuronames:756 {source="BIRNLEX:2638"}
 xref: UMLS:C0262207 {source="BIRNLEX:2638"}
+xref: ZFA:0000518
```

Comment:
Could it be that ZFA is not correct? [Olson et al.,](https://pmc.ncbi.nlm.nih.gov/articles/PMC6084820/) show that the 'Griseum centrale' is an homologue of the 'periaqueductal gray', which is found in the midbrain (and it is a synonym of 'central gray substance of midbrain' UBERON:0003040). This is also stated by [Blasiak et al](https://www.frontiersin.org/journals/molecular-neuroscience/articles/10.3389/fnmol.2022.984524/full). Therefore, I think that the maping is correct, but the ZFA textual and logical definition are not.

Reply:
> I think at this point, regardless of where the 'griseum centrale' really is in Zebrafish (hindbrain, as ZFA says, or midbrain?), the only sensible action for Uberon is to remove the mapping entirely. Better not to assert anything than to assert something dubious.

I agree, the literature is too conflictive in this case.